zerver/lib/markdown/help_settings_links.py

```diff
@@ -111,17 +111,18 @@
 
 1. Select **{setting_type_name}**.
 
-1. On the left, click {setting_reference}.
+1. On the left, click **{setting_reference}**.
 """
 
 
 def getMarkdown(setting_type_name: str, setting_name: str, setting_link: str) -> str:
-    if relative_settings_links:
-        return f"1. Go to [{setting_name}]({setting_link})."
-    return settings_markdown.format(
+    relative_link = f"[{setting_name}]({setting_link})"
+    relative_link_markdown = f"1. Go to {relative_link}."
+    setting_markdown = settings_markdown.format(
         setting_type_name=setting_type_name,
-        setting_reference=f"**{setting_name}**",
+        setting_reference=setting_name,
     )
+    return relative_link_markdown if relative_settings_links else setting_markdown
```

Comment:
Code style wise, I prefer multiple return statements if normal if/else clauses; it's easier for one's eyes to see the control code of the function. 

In this particular case the performance issue doesn't matter, but you're also computing both answers with this implementation, and that's something to avoid from a coding habits standpoint.